packages/feeds-client/src/Feed.ts

```diff
@@ -1112,7 +1119,11 @@ export class Feed extends FeedApi {
       this.state.next((currentState) => ({
         ...currentState,
         members: currentState.members
-          ? uniqueArrayMerge(currentState.members, members, ({ user }) => user.id)
+          ? uniqueArrayMerge(
+              currentState.members,
+              members,
+              ({ user }) => user.id,
+            )
```

Comment:
Not sure why this has changed, there's literally no code change here so I'd expect the formatting to stay the same.